README.md

```diff
@@ -15,49 +15,101 @@ For example, to enable debug logging for code from this repo but error logging f
 
     $ RUST_LOG=ERROR,ceramic_kubo_rpc=DEBUG,ceramic_one=DEBUG cargo run -p ceramic-one -- daemon
 
-## Packaging
-To package rust-ceramic, you will need the following dependencies
+## Installation
```

Comment:
Installation should go above usage?

Also, once installed I assume usage doesn't need to include cargo?

Reply:
moved usage below installation and updated to use the binary directly. also added some info about available flags